exercises/practice/nucleotide-count/.approaches/introduction.md

```diff
@@ -25,7 +24,7 @@ function count_nucleotides(strand)
 end
 ```
 
-## Using four variables
+## Approach: using if/elseif
```

Comment:
```suggestion
## Approach: using a variable for each nucleotide
```

I think this is the distinctive part of this approach, rather than the if/elseif